include/util.h

```diff
@@ -26,6 +26,9 @@
 /** No entry can be found with the given key from the associative data structures. */
 #define ERR_NODATA      (-5)
 
+/** Invalid argument to the interface. */
```

Comment:
Since we have ERR_IDX, which means the "invalid argument to index the container",
ERR_ARG maybe too general for precise error code specification.

Thus, ERR_GET maybe more appropriate. With the specification "Invalid argument to store the returned data from
getter operation".